<launch>
 <!-- Start Nodes -->
 <node name="joint_control" pkg="rpi_huno" type="joint_control" args="" />
</launch>
